src/lib/types.ts

```diff
@@ -529,7 +525,7 @@ export type DeleteFactorParams = {
 }
 
 export interface GoTrueMFAApi {
-  verify(params: MFAVerifyParams): Promise<AuthMFAResponse>
+  verify(params: MFAVerifyParams): Promise<AuthResponse>
```

Comment:
Might make sense to standardise all methods to either `AuthMFAResponse` or `AuthResponse`. Should we update AuthResponse and change the rest of the methods to AuthResponse instead as well? I can do it in a separate PR.